### PR TITLE
Update WebHook doc

### PR DESCRIPTION
**Pull request by valouvaliavlo**

### CODE REVIEW

**Comment by valouvaliavlo** [comment]
How can i update image ?

**Comment by Herklos** [comment]
> How can i update image ?

You should have a look at https://github.com/Drakkar-Software/OctoBot/tree/assets

File: docs/source/pages/Using-a-webhook-with-OctoBot.rst [hunk 1]
```diff
@@ -11,7 +11,11 @@ Setting up your OctoBot's webhook
 
 
 #. In OctoBot configuration, add the webhook service.
-#. To get your OctoBot's webhook address (the url other websites will use to send messages to your OctoBot), create an account on https://ngrok.com/ and enter your ngrok token into your webhook service configuration.
+#. To setting up you webhook through internet
```
Comment (Herklos) [inline docs/source/pages/Using-a-webhook-with-OctoBot.rst:14]:
```suggestion
#. To set up you webhook through internet
```

File: docs/source/pages/Using-a-webhook-with-OctoBot.rst [hunk 2]
```diff
@@ -11,7 +11,11 @@ Setting up your OctoBot's webhook
...
+
+   * Option 1: If your octobot is not exposed to internet you have to enable ngrok and create an account on https://ngrok.com/ and enter your ngrok token into your webhook service configuration.
```
Comment (Herklos) [inline docs/source/pages/Using-a-webhook-with-OctoBot.rst:16]:
```suggestion
   * Option 1: If your OctoBot is not exposed to the Internet you have to : 
    - enable ngrok
    - create an account on https://ngrok.com/
    - enter your ngrok token into your OctoBot's webhook service configuration.
```

File: docs/source/pages/Using-a-webhook-with-OctoBot.rst [hunk 3]
```diff
@@ -11,7 +11,11 @@ Setting up your OctoBot's webhook
...
+   * Option 2: If your octobot is exposed to internet you can disable ngrok and set the listening port and ip for the webhook
```
Comment (Herklos) [inline docs/source/pages/Using-a-webhook-with-OctoBot.rst:17]:
```suggestion
   * Option 2: If your OctoBot is exposed to the Internet you can disable ngrok and set the listening port and ip for the webhook
```

File: docs/source/pages/Using-a-webhook-with-OctoBot.rst [hunk 4]
```diff
@@ -26,4 +30,4 @@ Setting up your OctoBot's webhook
       :alt: webhook log
 
    ## About ngrok.com
-   You can use ngrok with a free account, the only drawback of having a free version is that your webhoob address will change at every OctoBot restart, you will have to update it on your message sender (like https://www.tradingview.com)
+   You can use ngrok with a free account, the only drawback of having a free version is that your webhook address will change at every OctoBot restart, you will have to update it on your message sender (like https://www.tradingview.com)
```
Comment (Herklos) [inline docs/source/pages/Using-a-webhook-with-OctoBot.rst:33]: :+1:

File: docs/source/pages/Using-a-webhook-with-OctoBot.rst (hunk 2 above)
Comment (valouvaliavlo) [reply docs/source/pages/Using-a-webhook-with-OctoBot.rst:16]: 👍

**Review by Herklos — APPROVED** [review]

**Comment by Herklos** [comment]
Thanks @valouvaliavlo !

File: docs/source/pages/Using-a-webhook-with-OctoBot.rst [hunk 5]
```diff
@@ -11,7 +11,14 @@ Setting up your OctoBot's webhook
 
 
 #. In OctoBot configuration, add the webhook service.
-#. To get your OctoBot's webhook address (the url other websites will use to send messages to your OctoBot), create an account on https://ngrok.com/ and enter your ngrok token into your webhook service configuration.
+#. To set up you webhook through internet
```
Comment (GuillaumeDSM) [inline docs/source/pages/Using-a-webhook-with-OctoBot.rst:14]:
```suggestion
#. To set up your webhook configuration
```

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**